test/router-luatest/router_test.lua

```diff
@@ -469,24 +469,23 @@ g.test_enable_disable = function(g)
         _G.fiber_new:set_joinable(true)
     end, {global_cfg})
 
-    -- emulate unconfigured box
-    router:exec(function()
-        rawset(_G, 'old_box_cfg', box.cfg)
+    local err1, err2 = router:exec(function()
+        -- emulate unconfigured box
+        local old_box_cfg = box.cfg
         box.cfg = function(...) return _G.old_box_cfg(...) end
```

Comment:
You have to change this line to use `old_box_cfg`, not `_G.old_box_cfg`.

Reply:
My bad( Fixed